yuca/ca/continuous.py

```diff
@@ -44,7 +44,7 @@ class CCA(CA):
     """
 
     def __init__(self, **kwargs):
-        super(CCA, self).__init__()
+        super(CCA, self).__init__(**kwargs)
```

Comment:
Important to pass on kwargs to parent class, particularly when non-default args are used.